test/testdata/container_config_folder_perm.json

```diff
@@ -0,0 +1,57 @@
+{
+	"metadata": {
+		"name": "folder-perm",
+		"attempt": 1
+	},
+	"image": {
+		"image": "quay.io/sohankunkerkar/etc-permission:latest"
```

Comment:
we can move this image under `quay.io/crio`

Reply:
instead of having a new json file, can you use jq to generate this?